website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -1106,6 +1106,12 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
+		vector_info: {
+			description:       "Has a fixed value of 1.0. Contains build information such as rust and vector versions."
+			type:              "gauge"
+			default_namespace: "vector"
+			tags:              _internal_metrics_tags
```

Comment:
I think you'll want to outline the exact tags added here (`rust_version`, `arch`, etc.) plus the normal `_internal_metrics_tags`.

```
tags:              _internal_metrics_tags & {
  // more tags
}
```